x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/trusted_apps_page.test.tsx

```diff
@@ -70,6 +74,17 @@ describe('When on the Trusted Apps Page', () => {
           per_page: httpOptions?.query?.per_page ?? 20,
         };
       }
+
+      if (PACKAGE_POLICY_API_ROUTES.LIST_PATTERN) {
```

Comment:
Umm, are you missing some === or match here?

Reply:
Yeah, I am. I fixed this already. Thanks for catching it.